src/Microsoft.AspNetCore.Authentication.OpenIdConnect/OpenIdConnectOptions.cs

```diff
@@ -55,6 +56,24 @@ public OpenIdConnectOptions(string authenticationScheme)
             Events = new OpenIdConnectEvents();
             Scope.Add("openid");
             Scope.Add("profile");
+
+            ClaimMaps.DeleteClaim("nonce");
```

Comment:
That's what overloads are for :)  I'd have the params method for this usage alone personally...